packaging/packager_pip.sh

```diff
@@ -43,7 +43,7 @@ pip3 install -r "$distro_root/guibot/packaging/pip_requirements.txt"
 # auto-review and linters
 cd "$distro_root/guibot"
 echo "Performing auto-review linting checks on code and documentation"
-pycodestyle guibot/* --ignore="E501,W503,E226,E265,E731,E306"
+pycodestyle guibot/* --ignore="E501,W503,E226,E265,E731,E306,E203"
```

Comment:
Why is E203 ignored? The CI is passing on master so the linters must be happy with all current python code and the current pull request doesn't modify anything python related to need this extra error ignoring.

Reply:
This was related to other tests (local ones)

Reply:
Ok, so I assume the old selection of ignorables will be recovered now